### DISCUSSION TITLE
同时开启刷紧急委托和侵蚀一的时候有可能信标数据收集不满

### DISCUSSION THREAD

**Discussion by guoh064** [LmeSzinc/AzurLaneAutoScript#3054] [General]
@SaiCateDoan 今天看运行发现的情况：

> 因为开启了侵蚀一，所以短猫在1000体力以下不会主动运行，会交给短猫；
> 而刷紧急委托脸黑一直没停，所以侵蚀一一直没跑。
> 当天大世界每日就140信标数据，这差点导致当天第二个信标没开的下来。（截至服务器16点，我后来手动刷了60点信标数据）

是否需要考虑增加一个判断来处理刷紧急委托占用过多的情况？

**Comment by AkiraChisaka** [comment]
这个是不是发在Issues里比较好？

感觉Issues版比这边更活跃的样子。
